index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>First Portfoloio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+    integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;300;400;700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header class="header">
+    <div class="left">
+      <a href="#">
+        <div class="logo">
+            My logo
+        </div>
+    </a>
+    </div>
+    <div class="right">
+        <div class="menubar">
+            <div class="menubar1"></div>
+            <div class="menubar2"></div>
+            <div class="menubar3"></div>
+        </div>
+    </div>
+</header>
+<section class="container">
+  <div class="name">
+    <h1>I’m Hafiz.</h1>
+    <span>Glad to see you!</span>
+</div>
+<div class="bio">
+  <p>I’m a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+</div>
+
+        <div class="lets_contact">
+          <div class="contact">
+            <h4>LET’S CONNECT</h4>
```

Comment:
- Good job using <hx>, however you should use h4 only if you already have h3, so kindly replace it with h2 :recycle: